var/spack/repos/builtin/packages/py-fsspec/package.py

```diff
@@ -12,7 +12,11 @@ class PyFsspec(PythonPackage):
     homepage = "https://github.com/intake/filesystem_spec"
     url      = "https://pypi.io/packages/source/f/fsspec/fsspec-0.4.4.tar.gz"
 
+    import_modules = ['fsspec', 'fsspec.implementations']
+
+    version('0.7.3', sha256='1b540552c93b47e83c568e87507d6e02993e6d1b30bc7285f2336c81c5014103')
     version('0.4.4', sha256='97697a46e8bf8be34461c2520d6fc4bfca0ed749b22bb2b7c21939fd450a7d63')
 
     depends_on('python@3.5:', type=('build', 'run'))
+    depends_on('python@3.6:', type=('build', 'run'), when='@0.6.3:')
```

Comment:
I just checked 0.7.3 and it still has `python_requires=">3.5"`, so I would remove this line.

Reply:
`>3.5` is `@3.6:`, isn't it?

Reply:
You are correct, my bad